docs/docs/guides/dbt-semantic-layer.mdx

```diff
@@ -3,77 +3,36 @@ import DbtJobUrl from './assets/dbt-job-url-example.png';
 
 # dbt Semantic Layer
 
-By integrating Lightdash with your dbt Semantic Layer, you can consume dbt metrics in real time and you
-can be sure
-you're calculating the exact metrics you see in your other semantic layer-connected tools.
+Lightdash is dbt Semantic Layer ready and the integration is now available in public preview for Lightdash Cloud users. By integrating Lightdash with your dbt Semantic Layer, you can consume dbt metrics in real time and you can be sure you're calculating the exact metrics you see in your other semantic layer-connected tools.
```

Comment:
> Lightdash is dbt Semantic Layer ready

note: My brain struggled to understand this part of the sentence because it was pausing before "ready"
"Lightdash is dbt Semantic Layer ... ready"